docs/shared/register-federated-cli.mdx

```diff
@@ -1,53 +1,42 @@
-In a federated architecture, each of your graph's [implementing services](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) uses the Apollo CLI to register its schema as a separate service **of the same graph**:
+In a federated architecture, each of your graph's [subgraphs](https://www.apollographql.com/docs/federation/implementing-services/) uses the Rover CLI to register its schema with Apollo:
 
 ```mermaid
-graph BT;
+graph LR;
   subgraph " "
   gateway([Gateway]);
-  serviceA[Products service];
-  serviceB[Reviews service];
-  serviceC[Inventory service];
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  serviceC[Inventory subgraph];
   end
   registry{{Apollo Schema Registry}};
   gateway --- serviceA & serviceB & serviceC;
-  serviceA -. --serviceName=Products  .-> registry;
-  serviceB -. --serviceName=Reviews  .-> registry;
-  serviceC -. --serviceName=Inventory  .-> registry;
+  serviceA & serviceB & serviceC -. rover subgraph publish  .-> registry;
   class registry secondary;
 ```
 
-First, <a href="https://www.apollographql.com/docs/devtools/cli/" target="_blank">install the Apollo CLI</a> if you haven't yet.
+If you haven't yet:
 
-Then, **do the following for each of your implementing services**:
+*  <a href="https://www.apollographql.com/docs/rover/getting-started/" target="_blank">Install the Rover CLI.</a>
+* <a href="https://www.apollographql.com/docs/rover/configuring/" target="_blank">Authenticate Rover</a> with Apollo Studio.
 
-1. Obtain the following values, which are required for the `apollo service:push` command:
+Then, **do the following for each of your subgraphs**:
 
-    * The URL that your federated gateway will use to communicate with the service (e.g., `http://products-graphql.svc.cluster.local:4001/`).
-    * The name that uniquely identifies the service within your graph (e.g., `products`).
-    * The service's schema. The Apollo CLI can either:
+1. Obtain the following values, which are required for the `rover subgraph publish` command:
+
+    * The name that uniquely identifies the subgraph within your graph (e.g., `products`).
```

Comment:
We may want to be more clear about where to find this -- and help people understand that they should not use a name already in use (this is an area where our system is more forgiving that we'd probably like, so giving people some guidance on this will be important).